build.gradle

```diff
@@ -15,6 +15,8 @@ buildscript {
     }
 }
 
+throw new GradleException('Do Not Build Witchery - source is for reference only!')
```

Comment:
Nice idea but I think just removing everything related to gradle maybe is better since you can't just remove one line and poof it builds again, so only the source and the README is left

Reply:
And removing the libs folder aswell, it's not to have it anyway

Reply:
left only that line, removed libs